benchmark/common.js

```diff
@@ -4,24 +4,24 @@ const child_process = require('child_process');
 const http_benchmarkers = require('./_http-benchmarkers.js');
 
 class Benchmark {
-  // Used to make sure a benchmark only start a timer once
-  #started = false;
+  constructor(fn, configs, options = {}) {
+    // Used to make sure a benchmark only start a timer once
+    this._started = false;
 
-  // Indicate that the benchmark ended
-  #ended = false;
+    // Indicate that the benchmark ended
+    this._ended = false;
```

Comment:
Nit: we can use symbol properties instead of the underscore prefix but I'm fine either way.

Reply:
It should be fine doing this for the benchmark tool, it can be reverted back anyway and replacing _ with # would be easier.